src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -20,6 +20,27 @@
 namespace chip {
 namespace scenes {
 
+namespace {
+
+/// ValideAttribute
+/// @brief  Validate the attribute exists for a given cluster
+/// @param[in] clusterID  Cluster ID
+/// @param[in] attID      Attribute ID
+/// @return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE if the attribute does not exist for a given cluster or is not scenable
+// TODO: Assess if we also want to throw an error if the attribute value is out of range
+// TODO: Add check for "S" quality to determine if the attribute is scenable once suported :
+// https://github.com/project-chip/connectedhomeip/issues/24177
+CHIP_ERROR ValideAttribute(EndpointId endpoint, ClusterId cluster, AttributeId attributeId)
```

Comment:
Note that this will return error for global list attributes.  That's probably OK, since those are not scenable anyway....

Reply:
I'll add a note there just in case we someday want to make global attributes scenable.